searx/settings.yml

```diff
@@ -741,6 +741,23 @@ engines:
       require_api_key: false
       results: HTML
 
+  - name : azlyrics
+    shortcut : lyrics
```

Comment:
I am a bit on the fence with this shortcut because it is not future-proof. We haven't got any lyrics search engines, but if we get some in the future it would look a bit weird to have one engine with this shortcut.  Maybe `azl` would be a better shortcut.

Reply:
You are right, I changed it now.